DevelopGitHub/GitHubExtension/GitHubExtension.Infrastructure.Constants/RouteConstants.cs

```diff
@@ -9,6 +9,8 @@ public static class RouteConstants
         public const string ApiUser = "api/user";
 
         // static segments
+        public const string Home = "http://localhost:50859/";
```

Comment:
As I understand it will be work correctly if we deploy our application on the server. Correct?